vignettes/adtr.Rmd

```diff
@@ -0,0 +1,734 @@
+---
+title: "Creating ADTR"
+output: 
+  rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{Creating ADTR}
+  %\VignetteEncoding{UTF-8}
+  %\VignetteEngine{knitr::rmarkdown}
+---
+
+```{r setup, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+library(admiral)
+link <- function(text, url) {
+  return(
+    paste0(
+      "[", text, "]",
+      "(", url, ")"
+    )
+  )
+}
+dyn_link <- function(text,
+                     base_url,
+                     relative_url = "",
+                     # Change to TRUE when admiral adopts multiversion docs
+                     is_multiversion = FALSE,
+                     multiversion_default_ref = "main") {
+  url <- paste(base_url, relative_url, sep = "/")
+  if (is_multiversion) {
+    url <- paste(
+      base_url,
+      Sys.getenv("BRANCH_NAME", multiversion_default_ref),
+      relative_url,
+      sep = "/"
+    )
+  }
+  return(link(text, url))
+}
+# Other variables
+admiral_homepage <- "https://pharmaverse.github.io/admiral"
+library(admiraldev)
+```
+
+# Introduction
+
+This article describes creating an `ADTR` (Tumor Results) ADaM with common
+oncology parameters based on RECIST v1.1.
+
+The main part in programming a tumor results dataset is the calculation of the
+sum of diameters of all target lesions (lymph nodes & non-lymph nodes), the
+calculation of nadir, change & percentage change from baseline, and the
+analysis flags that could be required for reporting.  The tumor results data could be set up for investigator and/or Independent review facility (IRF)/Blinded Independent Central Review (BICR) data.  The below sample code would need to be updated (for example, update the Evaluator `TR.TREVAL`, `TU.TUEVAL`, and the applicable parameter details `PARAM`, `PARAMCD`, `PARCATx`) in order to create the acquired data for Independent review facility (IRF)/Blinded Independent Central Review (BICR).
```

Comment:
appear to be a few erroneous line breaks here (line 52, 53 and 54)

Reply:
It doesn't matter if you have line breaks within a paragraph or if the whole paragraph is written in one line. On the website it is displayed correctly.